src/pages/proposals/comment/author.js

```diff
@@ -30,19 +32,27 @@ export default class CommentAuthor extends React.Component {
         <span data-digix="CommentAuthor-QuarterPoints">
           {UserStats.quarterPoints}: {user.quarterPoint}
         </span>
+        {createdAt && (
```

Comment:
I don't think the `createdAt &&` checker necessary here since comments have this data already by default. We also don't need to have a `defaultProp` for this for the same reason (if `props.user` is required, it should follow that the comment will also have `createdAt` already anyway).

Reply:
Yea you are right, i think i wanted to be extra careful. I didnt have a proper setup when i create this PR. Now Rohit has helped me to setup everything :D. I wont have so much time in the next few days. Will fix the comments in the PRs in the next few days :).